api/envoy/api/v2/discovery.proto

```diff
@@ -5,6 +5,7 @@ package envoy.api.v2;
 import "envoy/api/v2/core/base.proto";
 
 import "google/protobuf/any.proto";
+import "google/protobuf/duration.proto";
```

Comment:
@htuch presumably we should add this to v3 only, right? I assume the v2 freeze also includes the transport protocol?

Reply:
Yep, no major new features in v2 transport I think.

Reply:
removed the v2 changes